tests/includes/legacy/importer/tests-donations.php

```diff
@@ -71,6 +71,12 @@ class Tests_Give_Import_Donations extends Give_Unit_Test_Case {
 	 */
 	protected $total = '';
 
+    /**
+     * @unreleased
+     * @var string
+     */
+    protected $main_key = '';
```

Comment:
I removed this a while back because it was causing an error... I'm really confused. 😅

Reply:
I do not know what to say 🤔

Reply:
Well... I guess since the tests are passing it's fine. 🤷‍♂️